tests/test_bitfield.py

```diff
@@ -253,22 +253,22 @@ def test_bitfield_char(compiled):
     struct test {
         uint16  a : 4;
         uint16  b : 4;
-        char    c : 4;
+        char    c : 8;
         char    d[4];
     };
     """
 
     cs = cstruct.cstruct()
-    cs.load(cdef)
+    cs.load(cdef, compiled=compiled)
 
-    assert verify_compiled(cs.test, compiled)
+    assert verify_compiled(cs.test, compiled=compiled)
```

Comment:
```suggestion
    assert verify_compiled(cs.test, compiled)
```
Just for consistency sake.